vignettes/articles/oauth.Rmd

```diff
@@ -56,94 +75,159 @@ client <- oauth_client(
 )
 ```
 
-You need to figure out the `token_url` from the [documentation](https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps).
-I wish I could give good advice about how to find it 😞.
+You can certainly uncover my client secret if you are an experienced R programmer and are willing to spend a bit of time experimenting, but I'm pretty sure it'd be easy for you to just create your own app on GitHub.
+
+### Packaging
 
-Note that if you print the client the secret is automatically redacted:
+I recommend wrapping client creation in a function in your package, e.g.:
```

Comment:
Should there be a sentence at the beginning explaining you assume most docs readers will be packaging their OAuth code but that the vignette still applies if you're writing a script?